util/dir/src/lib.rs

```diff
@@ -226,16 +226,26 @@ impl DatabaseDirectories {
 	}
 }
 
+fn default_path(t: AppDataType, product_name: &'static str) -> Option<PathBuf> {
+	let app_info = AppInfo { name: product_name, author: AUTHOR };
+	let old_root = get_app_root(t, &app_info).ok()?;
+	if old_root.exists() {
+		return Some(old_root);
+	}
+
+	let mut root = data_root(t).ok()?;
+	root.push(if LOWERCASE { "openethereum" } else { "OpenEthereum" });
```

Comment:
why not use lowercase everywhere?

Reply:
I think the convention for `AppData` and `Application Support` is to use the original case.